components/account/AccountMoreButton.vue

```diff
@@ -23,7 +23,7 @@ const { client } = $(useMasto())
 const useStarFavoriteIcon = usePreferences('useStarFavoriteIcon')
 const { share, isSupported: isShareSupported } = useShare()
 
-function recordEngagement(dataGlean) {
+function recordEngagement(dataGlean: string) {
```

Comment:
Added a type here to make vscode happy

Reply:
Hmm my vscode doesn't get sad over this. Do you have any idea what the difference is in our IDEs?

Reply:
Not sure. I thought maybe I had a typescript extension on in vscode, but apparently I don't. Do you have one?

Reply:
I disabled some extension with the word TypeScript in it that I couldn't find in the marketplace, and then I updated my IDE. Now I'm getting the error. Not sure which of the two was the culprit though